src/components/themed/ControlPanel.tsx

```diff
@@ -347,7 +347,13 @@ export function ControlPanel(props: DrawerContentComponentProps) {
                   <TitleText style={styles.text}>{username}</TitleText>
                 </TouchableOpacity>
                 <TouchableOpacity style={styles.rowIconContainer} onPress={handleDeleteAccount(username)}>
-                  <MaterialIcon size={theme.rem(1.5)} name="close" color={theme.iconTappable} />
+                  <MaterialIcon
+                    size={theme.rem(1.5)}
+                    name="close"
+                    color={theme.iconTappable}
+                    accessibilityHint={lstrings.close_control_panel_hint}
+                    testID={lstrings.close_control_panel_hint}
```

Comment:
When we break props over multiple lines, can we alphabetize them? VSCode has a "Sort lines ascending" button that's good for this.